docs/backend-system/core-services/root-http-router.md

```diff
@@ -27,11 +27,11 @@ createBackendPlugin({
       },
       async init({ rootHttpRouter }) {
         const router = Router();
-        router.get('/health', (request, response) => {
+        router.get('', (request, response) => {
```

Comment:
Maybe we should remove the `''` entirely, or use `*` or something a little better. It's a little confusing IMO in the docs to just have an empty string.

Alternatively, we could use something like `/readiness` or `/liveness` as they're common patterns in k8s to have different paths for different healthchecks.

Reply:
Went with `/health/readiness` to make it more explicit :)